test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Export/ExportTestHelper.cs

```diff
@@ -63,14 +63,69 @@ internal static async Task<IList<Uri>> CheckExportStatus(TestFhirClient testFhir
             return exportJobResult.Output.Select(x => x.FileUri).ToList();
         }
 
-        internal static async Task<Dictionary<(string resourceType, string resourceId), Resource>> GetResourcesFromFhirServer(
+        internal static async Task<Dictionary<(string resourceType, string resourceId, string versionId), Resource>> GetResourcesFromFhirServer(
             TestFhirClient testFhirClient,
             Uri requestUri,
             FhirJsonParser fhirJsonParser,
             ITestOutputHelper outputHelper)
         {
-            var resourceIdToResourceMapping = new Dictionary<(string resourceType, string resourceId), Resource>();
+            var resourceIdToResourceMapping = new Dictionary<(string resourceType, string resourceId, string versionId), Resource>();
 
+            try
+            {
+                await foreach (Resource resource in GetResourceListFromFhirServer(testFhirClient, requestUri, fhirJsonParser))
+                {
+                    resourceIdToResourceMapping.TryAdd((resource.TypeName, resource.Id, resource.VersionId), resource);
+                }
+            }
+            catch (Exception ex)
+            {
+                outputHelper.WriteLine($"Unable to parse response into bundle: {ex}");
+                return resourceIdToResourceMapping;
```

Comment:
Nit: this technically isn't needed as it will fall through to the return on line 87.